src/providers/definition/common.ts

```diff
@@ -111,5 +112,15 @@ export const viewSuggestions: DefinitionSuggestion[] = [
 		files (document: TextDocument, text: string): string[] {
 			return [ document.fileName.replace(/app\/(.*)$/, `app/controllers/${text}.js`) ];
 		}
+	},
+	{ // i18n
+		regExp: /[:\s=,>)("]L\(["'][\w0-9_-]*$/,
+		definitionRegExp(text: string): RegExp {
+			return new RegExp(`name=["']${text}["']>(.*)?</`, 'g');
```

Comment:
<!--
  0 failure: 
  1 warning:  src/providers/def...
  
  
  DangerID: danger-id-Danger;
  File: src/providers/definition/common.ts;
  Line: 119;
-->

- :warning: src/providers/definition/common.ts line 119 – Found non-literal argument to RegExp Constructor (security/detect-non-literal-regexp)